maintainers/maintainer-list.nix

```diff
@@ -2375,6 +2375,12 @@
     githubId = 10913120;
     name = "Dje4321";
   };
+  djwf = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Both are done.